shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/dal/MySQLResetStatement.java

```diff
@@ -17,14 +17,21 @@
 
 package org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.dal;
 
+import lombok.Getter;
 import lombok.ToString;
 import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
 import org.apache.shardingsphere.sql.parser.sql.common.statement.dal.DALStatement;
 import org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.MySQLStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.segment.ResetOptionSegment;
+
+import java.util.LinkedList;
+import java.util.List;
 
 /**
  * MySQL reset statement.
  */
 @ToString
+@Getter
 public final class MySQLResetStatement extends AbstractSQLStatement implements DALStatement, MySQLStatement {
+    private List<ResetOptionSegment> options = new LinkedList<>();
```

Comment:
@flycash Please add a blank line before this line.

Reply:
done